python_modules/libraries/dagster-components/dagster_components/core/component.py

```diff
@@ -33,7 +36,34 @@ def from_decl_node(
     ) -> Self: ...
 
 
+def get_entry_points(group: str) -> Sequence[importlib.metadata.EntryPoint]:
```

Comment:
It would nice if `get_entry_points` was named more clearly to indicate that it is searching all the modules loaded in scope.